# Problem Statement 1:

In [32]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
%matplotlib inline
import scipy.stats as stats


Calculate the mean, median, mode and standard deviation for the problem statements 1& 2.

The marks awarded for an assignment set for a Year 8 class of 20 students were as
follows:
6 7 5 7 7 8 7 6 9 7 4 10 6 8 8 9 5 6 4 8

In [3]:
df = pd.DataFrame({'Year8' : [6, 7, 5, 7, 7, 8, 7, 6, 9, 7, 4, 10, 6, 8, 8, 9, 5, 6, 4, 8]})
df.head()

Year8
0      6
1      7
2      5
3      7
4      7

# Mean

In [4]:
df.mean()

Year8    6.85
dtype: float64

# Median

In [5]:
df.median()

Year8    7.0
dtype: float64

# Mode

In [6]:
df.mode()

Year8
0      7

# Standard Deviation

In [9]:
df.std().round(2)

Year8    1.63
dtype: float64

# Problem Statement 2

Problem Statement 2:
The number of calls from motorists per day for roadside service was recorded for a
particular month:

28, 122, 217, 130, 120, 86, 80, 90, 140, 120, 70, 40, 145, 113, 90, 68, 174, 194, 170,
100, 75, 104, 97, 75,
123, 100, 75, 104, 97, 75, 123, 100, 89, 120, 109

In [10]:
df2 = pd.DataFrame({'Calls' : [28, 122, 217, 130, 120, 86, 80, 90, 140, 120, 70, 40, 145, 113, 90, 68, 174, 194, 170,
100, 75, 104, 97, 75,
123, 100, 75, 104, 97, 75, 123, 100, 89, 120, 109]})
df2.head()

Calls
0     28
1    122
2    217
3    130
4    120

# Mean

In [12]:
df2.mean().round(2)

Calls    107.51
dtype: float64

# Mode

In [13]:
df2.mode().round(2)

Calls
0     75

# Standard Deviation

In [15]:
df2.std().round(2)

Calls    39.34
dtype: float64

# Median

In [21]:
df2['Calls'].median().round(2)

100.0

# Problem Statement 3

 ` The number of times I go to the gym in weekdays, are given below along with its
 associated probability:
 x = 0, 1, 2, 3, 4, 5
 f(x) = 0.09, 0.15, 0.40, 0.25, 0.10, 0.01
 Calculate the mean no. of workouts in a week. Also evaluate the variance involved in it`

In [24]:
df3 = pd.DataFrame({'x': [0,1,2,3,4,5] , 'f(x)':[0.09, 0.15, 0.40, 0.25, 0.10, 0.01]})
df3.head()

x  f(x)
0  0  0.09
1  1  0.15
2  2  0.40
3  3  0.25
4  4  0.10

# Mean

In [28]:
df3['f(x)'].mean().round(2)

0.17

In [31]:
df3['f(x)'].var().round(3)

0.019

# Problem Statement 4
Let the continuous random variable D denote the diameter of the hole drilled in an aluminum sheet. The target diameter to be achieved is 12.5mm. Random disturbances in the process often result in inaccuracy.

Historical data shows that the distribution of D can be modelled by the PDF (𝑑) = 20𝑒−20(𝑑−12.5), 𝑑 ≥ 12.5. If a part with diameter > 12.6 mm needs to be scrapped,

what is the proportion of those parts? What is the CDF when the diameter is of 11 mm? What is your conclusion regarding the proportion of scraps?

P = (12.5 < x < 12.6) = 1 - P(x > 12.6) = 1 - 0.135 =  0.865

# Problem Statement 5:

A company manufactures LED bulbs with a faulty rate of 30%. If I randomly select 6
chosen LEDs, what is the probability of having 2 faulty LEDs in my sample?
Calculate the average value of this process. Also evaluate the standard deviation
associated with it.

# Problem Statement 6:
Gaurav and Barakha are both preparing for entrance exams. Gaurav attempts to
solve 8 questions per day with a correction rate of 75%, while Barakha averages
around 12 questions per day with a correction rate of 45%. What is the probability
that each of them will solve 5 questions correctly? What happens in cases of 4 and 6
correct solutions? What do you infer from it? What are the two main governing
factors affecting their ability to solve questions correctly? Give a pictorial
representation of the same to validate your answer.

# Problem Statement 7:
Customers arrive at a rate of 72 per hour to my shop. What is the probability of 𝑘
customers arriving in 4 minutes? a) 5 customers, b) not more than 3 customers, c)
more than 3 customers. Give a pictorial representation of the same to validate your


# Problem Statement 8:


I work as a data analyst in Aeon Learning Pvt. Ltd. After analyzing data, I make
reports, where I have the efficiency of entering 77 words per minute with 6 errors per
hour. What is the probability that I will commit 2 errors in a 455-word financial report?
What happens when the no. of words increases/decreases (in case of 1000 words,
255 words)?


How is the 𝜆 affected?

How does it influence the PMF?

Give a pictorial representation of the same to validate your answer.

# Problem Statement 9:


mm?
What is the conclusion of this experiment?
Customers arrive at a rate of 72 per hour to my shop. What is the probability of 𝑘 customers arriving in 4 minutes? 
a) 5 customers, 
b) not more than 3 customers, 
c) more than 3 customers. Give a pictorial representation of the same to validate your answer.

I work as a data analyst in Aeon Learning Pvt. Ltd. After analyzing data, I make reports, where I have the efficiency of entering 77 words per minute with 6 errors per hour. What is the probability that I will commit 2 errors in a 455-word financial report?

What happens when the no. of words increases/decreases (in case of 1000 words, 255 words)?
Let the continuous random variable D denote the diameter of the hole drilled in an aluminum sheet. The target diameter to be achieved is 12.5mm. Random disturbances in the process often result in inaccuracy.

Historical data shows that the distribution of D can be modelled by the PDF, f(d) = 20e−20(d−12.5), d ≥ 12.5. If a part with diameter > 12.6 mm needs to be scrapped,
what is the proportion of those parts?

What is the CDF when the diameter is of 11

# Problem Statement 10:


Please compute the following:
a) P(Z > 1.26), . P(Z < −0.86), P(Z > −1.37), P(−1.25 < Z < 0.37), . P(Z ≤ −4.6)

b) Find the value 𝑧 such that 𝑃(𝑍 > 𝑧) = 0.05

c) Find the value of 𝑧 such that 𝑃(−𝑧 < 𝑍 < 𝑧) = 0.99

# Problem Statement 11:


Problem Statement 11:
 The current flow in a copper wire follow a normal distribution with a mean of 10 𝑚A
 and a variance of 4 (𝑚𝐴)2.
    
 What is the probability that a current measurement will exceed 13 𝑚𝐴? 

 What is the probability that a current measurement is between 9 and 11mA? 
 
 Determine the current measurement which has a probability of 0.98.

# Problem Statement 12:


The shaft in a piston has its diameter normally distributed with a mean of 0.2508 inch and a standard deviation of 0.0005 inch. The specifications of the shaft are 0.2500 ∓
0.0015 inch. 

What proportion of shafts are in sync with the specifications? 

If the process is centered so that the mean is equal to the target value of 0.2500, what proportion of shafts conform to the new specifications? 

What is your conclusion from this experiment?